addon/mixins/axes.js

```diff
@@ -16,11 +16,65 @@ export default Ember.Mixin.create({
   graphicHeight: null,
   minXTicks: 3,
   minYTicks: 3,
-  tickSpacing: 50,
-
   minAxisValue: 0,
   maxAxisValue: 0,
 
+  /**
+   * We used to have only one option to set tick spacing for both x and y axes.
+   * We keep this attribute for backward compatibility.
+   * @type {number}
```

Comment:
Yeah, I've seen the syntax highlighting on here do all kinds of weird stuff. Not bad to consider, but probably more false-positives than anything.